docs/3.api/5.commands/devtools.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "nuxi devtools"
+description: The devtools command allows you to enable or disable Nuxt Devtools on a per-project basis.
+---
+
+# `nuxi devtools`
+
+```{bash}
+npx nuxi devtools enable|disable [rootDir]
+```
+
+Running `nuxi devtools enable` will install the Nuxt DevTools globally, and also enable it within the particular project you are using. It is saved as a preference in your user-level `.nuxtrc`. If you want to remove devtools support for a particular project, you can run `nuxi devtools disable`.
```

Comment:
```suggestion
Running `nuxi devtools enable` will install the Nuxt Devtools globally, and also enable it within the particular project you are using. It is saved as a preference in your user-level `.nuxtrc`. If you want to remove devtools support for a particular project, you can run `nuxi devtools disable`.
```

Reply:
We're using the `DevTools` casing for now.